Implement requests from PR#210 (#214)
sigstore/docs: content/en/contributing.md

category: "About sigstore"
position: 20
---

## Contributing as a developer

> You can contribute the documentation and you can also contribute as a developer. Check out the following repositories for developer information on the various Sigstore projects:

Reviewer: ```suggestion
To contribute to Sigstore as a developer, check out the following repositories for developer information on the various Sigstore projects:
```

Author: ok, accepted.

- And above all, be consistent. This applies both to the words you use, and the way you use them - so things like grammar and formatting, whether you capitalise a word or not, etc.

### Working with Nuxt

The Nuxt documentation is a great first stop for anyone new to writing technical documentation, or anyone interested in what Nuxt supports. All their guidance for writing, fetching and displaying content can be found here. It’s quick to pick up, and essential for transforming markdown files and raw text into well designed, readable guides and resources anyone can pick up.

Resource: https://content.nuxtjs.org/

> ## Contributing as a developer

Reviewer 2: Can this be moved up to the top? We'll have more contributing to these projects rather than only docs.

Author: paragraph moved to top of file.

Reviewer 2: @jonvnadelberg Can we move the "Contributing as a developer" section, not the Nuxt section?

Author: Yes.  I'll put the nuxt section back and move the contributing as a developer to the top.

Author: ok, the requested section has been moved.
